manta-js/package/src/utils.ts

```diff
@@ -63,4 +63,15 @@ export class MantaUtilities {
       console.error(e);
     }
   }
+
+  /// Creates a public transfer payload.
+  static async publicTransferBuild(api:ApiPromise, assetId: BN, amount: BN, destinationAddress: Address): Promise<any> {
+    const assetIdArray = Array.from(MantaPrivateWallet.assetIdToUInt8Array(assetId));
```

Comment:
Removed this function in favor of this: https://github.com/Manta-Network/sdk/blob/37e7a879850639c95b93fe993ab02c2a1d145020/manta-js/package/src/privateWallet.ts#L375